go.mod

```diff
@@ -35,10 +36,9 @@ require (
 	github.com/x-cray/logrus-prefixed-formatter v0.5.2
 	github.com/xeipuuv/gojsonschema v1.2.0
 	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
-	golang.org/x/net v0.0.0-20200625001655-4c5254603344 // indirect
-	golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208 // indirect
-	golang.org/x/sys v0.0.0-20200610111108-226ff32320da // indirect
-	google.golang.org/genproto v0.0.0-20200615140333-fd031eab31e7 // indirect
-	gopkg.in/ini.v1 v1.57.0 // indirect
 	gopkg.in/yaml.v2 v2.3.0
 )
+
+replace github.com/anchore/stereoscope => ../stereoscope
+
+replace github.com/anchore/go-testutils => ../go-testutils
```

Comment:
Q: Did we decide yesterday not to push `replace` lines?

Reply:
good question: are we saying not to push replace lines? or not push pinned commits that aren't on main? (or both?)

I feel that there is a lot of good content ready for review and there is 0-risk of merging a replace line since it cannot pass tests (unlike a pinned commit not on main).

I'd vote allowing pushing of replace lines but not pins of commits that are not on main.

cc: @alfredodeza

Reply:
from an offline conversation: I'll refrain from adding replace lines in commit log in the future